week1/Matrices.ipynb

```diff
@@ -0,0 +1,532 @@
+{
```

Comment:
<p>Also, talk about the applications/meaning of matrices and their operations (matrix-matrix, matrix-vector, determinant of matrix)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/toggle-corp/ml-internship/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>